src/lib/rest/rest.cpp

```diff
@@ -193,6 +195,23 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
       ciP->uriParamTypes.push_back(val);
     }
   }
+  else if (key == URI_PARAM_Q)
+  {
+    if (*val != 0)
```

Comment:
I think we don't need this parsing if at the end we move this to Scope (covering the query context case). Or maybe in the case of things already in Scope (like georel= ) we are doing also parsing a URI param level?